incfly/eds.yaml

```diff
@@ -0,0 +1,17 @@
+version_info: "0"
```

Comment:
I assume these are not going to be checked in..

Reply:
removed. superseded by the integration test.